app/src/app/data.service.ts

```diff
@@ -18,15 +23,23 @@ export class DataService {
   public splittedTraining: string[];
   public keyDetails: KeyDetails;
   public timelineRole: TimelineRole;
-
+  public capability: Capability;
+  public band: Band;
+  public family: Family;
+  public roles: Roles;
   public carouselRole: CarouselRole[] = [];
+
   constructor(private http: HttpClient) {
-    this.getUser();
-    this.getRoleSpecification();
+    this.getBand();
+    this.getCapability();
+    this.getCarouselRoleDetails();
+    this.getFamily();
     this.getKeyDetails();
+    this.getRoles();
+    this.getRoleSpecification();
     this.getTimelineRoles();
-    this.getCarouselRoleDetails();
-   }
+    this.getUser();
+  }
 
    public getUser(): void {
```

Comment:
your getters should return an `Observable`, getters dont return void, the clue is in the name

Reply:
This will be changed in #36